### PR TITLE
[POC][ResourceBundle] Form Factory

### PR DESCRIPTION
**Pull request by aRn0D**
@pjedrzejewski It is new kind of RFC. Is it enough readable? What do you think? \cc @Sylius/core-team

### CODE REVIEW

File: src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php [hunk 1]
```diff
@@ -342,20 +342,7 @@ public function getForm($resource = null)
     {
         $type = $this->config->getFormType();
 
-        if (strpos($type, '\\') !== false) { // full class name specified
-            $type = new $type();
```
Comment (Shine-neko) [inline src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php]: you plan to do something about this ?

Comment (amenophis) [inline src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php]: Used it really often, so need a similar functionnality after your refacto !

Comment (aRn0D) [inline src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php]: see the spec it_creates_an_instance

**Comment by aRn0D** [comment]
@pjedrzejewski What do you think?

**Comment by aRn0D** [comment]
@pjedrzejewski ping

**Comment by aRn0D** [comment]
@pjedrzejewski What do you think about the idae ?

**Comment by aRn0D** [comment]
@Sylius/core-team Any opinion ?

**Comment by aRn0D** [comment]
@pjedrzejewski ping ?

**Comment by pjedrzejewski** [comment]
I like the idea, however I do not like the deps on ObjectManager, perhaps we can introduce some kind of FormFieldsGuesser to make it abstract and replacable?

**Comment by aRn0D** [comment]
@pjedrzejewski I don't like it too. I need to find a solution !

**Comment by aRn0D** [comment]
@pjedrzejewski I update it but it not finish can you give me feedback, plz ?

**Comment by pjedrzejewski** [comment]
Looks good to me so far! :+1:

**Pull request closed without merging**